common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -296,9 +296,11 @@ def copy_or_rename_transcript(new_name, old_name, item, delete_old=False, user=N
 def get_html5_ids(html5_sources):
     """
     Helper method to parse out an HTML5 source into the ideas
-    NOTE: This assumes that '/' are not in the filename
+    NOTE: This assumes that '/' are not in the filename.
+    Slices each id by 100, restricting too long strings as video names.
     """
-    html5_ids = [x.split('/')[-1].rsplit('.', 1)[0] for x in html5_sources]
+    html5_ids = [x.split('/')[-1].rsplit('.', 1)[0][:100] for x in html5_sources]
```

Comment:
Is this line going to fix the current issue? It seems the JS change is required for prevention and backend change is being made for the fix?